core/src/test/java/feign/client/AbstractClientTest.java

```diff
@@ -332,6 +332,9 @@ public void testAlternativeCollectionFormat() throws Exception {
     @RequestLine("POST /?foo=bar&foo=baz&qux=")
     @Headers({"Foo: Bar", "Foo: Baz", "Qux: ", "Content-Type: {contentType}"})
     Response postWithContentType(String body, @Param("contentType") String contentType);
+
+    @RequestLine("GET /wsdl/testcase?wsdl")
+    Response getWsdl();
```

Comment:
This is a poor name for a test.  This does not indicate what the test actually is.  A more appropriate name would be `getRequestWithNameOnlyParameter()` or something similar.